packages/terra-clinical-application/src/Application.jsx

```diff
@@ -3,7 +3,6 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import Base from 'terra-base';
```

Comment:
Looks like terra-base is a hard dependency for terra-clinical-application. I've left this is to keep the PR passive.

Reply:
Talked with the clinical team, plan to leave this import in here as is